Assets/PatchKit/UnityEditorExtension/Editor/Api/Models/Keys/LicenseKey.cs

```diff
@@ -1,32 +1,26 @@
-using Newtonsoft.Json;
+using System;
 
 namespace PatchKit.Api.Models.Keys
 {
-    public struct LicenseKey
-    {
-        [JsonProperty("key")]
-        public string Key;
-        
-        [JsonProperty("app_secret")]
-        public string AppSecret;
-        
-        [JsonProperty("key_secret")]
-        public string KeySecret;
-        
-        [JsonProperty("collection_id")]
-        public int CollectionId;
-        
-        /// <summary>
-        /// Number of key registrations. This is a request wihout a app_secret.
-        /// </summary>
-        [JsonProperty("registrations")]
-        public int Registrations;
-        
-        /// <summary>
-        /// If set to true, this key is blocked for further use.
-        /// </summary>
-        [JsonProperty("blocked")]
-        public bool Blocked;
-        
-    }
+[Serializable]
+public struct LicenseKey
+{
+    public string key;
+
+    public string app_secret;
+
+    public string key_secret;
+
+    public int collection_id;
+
+    /// <summary>
+    /// Number of key registrations. This is a request wihout a app_secret.
+    /// </summary>
+    public int registrations;
+
+    /// <summary>
+    /// If set to true, this key is blocked for further use.
+    /// </summary>
+    public bool blocked;
```

Comment:
Missing tab indentation in the namespace region.

Reply:
done